cmd/vault-util/commands/root.go

```diff
@@ -0,0 +1,71 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/chef/ci-studio-common/cmd/vault-util/internal/pkg/account"
+	"github.com/chef/ci-studio-common/internal/pkg/paths"
+)
+
+var (
+	cfgFile string
+	rootCmd = &cobra.Command{
+		Use:          "vault-util",
+		Short:        "Utility to access secrets and account information stored in Hashicorp Vault from CI.",
+		SilenceUsage: true,
+	}
+
+	// These are shared across all the commands
+	accountCache = account.NewAccountCache()
+)
+
+// Execute handles the execution of child commands and flags
+func Execute() {
+	if rootCmd.Execute() != nil {
+		os.Exit(-1)
+	}
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+
+	// global config
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", fmt.Sprintf("configuration file (default is %s/vault-util.toml)", paths.SettingsDir))
+}
+
+// initConfig reads in config file and ENV variables if set.
+func initConfig() {
+	account.InitConfig()
+
+	viper.SetConfigName("vault-util")    // name of config file (without extension)
+	viper.AddConfigPath(paths.SettingsDir) // adding settings directory as first search path
+	viper.AddConfigPath(".")             // adding cwd directory as first search path
```

Comment:
I don't particularly care, but if you run `gofmt` on the code, it'll do little things like align these comments for you.